src/Data/Argonaut/Core.purs

```diff
@@ -185,8 +185,12 @@ foreign import fromBoolean :: Boolean -> Json
 -- | Construct `Json` from a `Number` value
 foreign import fromNumber :: Number -> Json
 
--- | Construct `Json` from a `String` value. If you would like to parse a string
--- | of JSON into valid `Json`, see `jsonParser`.
+-- | Construct `Json` from a `String` value.
```

Comment:
Would this be clearer if it said "Construct a `Json` string from a `String` value" to make it clear it's just producing a string, and not bonafide JSON with strings, numbers, objects, etc.?

Reply:
"`Json` string" makes me think it produces the string representation of the json contents - like what you'd get from `stringify`.
What about changing to: "Construct the `Json` representation of a `String` value"?
I feel like the output is still bona fide JSON, even if it's very simple JSON in this case.
Also, if we change that first line, we should probably change all the other `fromX` docs.

Reply:
I'm happy with that phrasing. And you're right, it's still bona fide JSON.